lirc_client.py

```diff
@@ -225,17 +226,14 @@ class AbstractLircClient:
          - verbose: boolean, if True print some progress info
     """
 
-    _socket = None
-    _verbose = False
-
-    _last_remote = None
-
     def __init__(self, verbose, timeout):
         self.timeout = timeout
         self.verbose = verbose
         self._parser = ReplyParser()
+        self._socket = None
         self._in_buffer = bytearray(0)
         self._last_command = None
+        self._last_remote = None
```

Comment:
Of course, the said variables were intended as instance variables, and it was an implementation error (not a design error)